From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Eneas U de Queiroz <cote2004-github@yahoo.com>
Date: Thu, 27 Sep 2018 08:34:38 -0300
Subject: Do not build the fuzz and documentation directories

This shortens build time.

Signed-off-by: Eneas U de Queiroz <cote2004-github@yahoo.com>

--- a/build.info
+++ b/build.info
@@ -4,3 +4,3 @@
-SUBDIRS=crypto ssl apps util tools fuzz providers doc
+SUBDIRS=crypto ssl apps util tools providers
 IF[{- !$disabled{tests} -}]
   SUBDIRS=test
